TASK/TaskVehicleDriveWander.md

```diff
@@ -10,8 +10,15 @@ void TASK_VEHICLE_DRIVE_WANDER(Ped ped, Vehicle vehicle, float speed, int drivin
 
 
 ## Parameters
-* **ped**: 
-* **vehicle**: 
-* **speed**: 
-* **drivingStyle**: 
+* **ped**: Ped id for the task.
+* **vehicle**: Vehicle entity id for the task.
+* **speed**: Speed of driving.
+* **drivingStyle**: Use 447 to simulate a real-life driver. More info can be found at [Here](https://vespura.com/fivem/drivingstyle/)
 
+Drive randomly with no destination set.
```

Comment:
Please move this line above the Parameters section for it to appear in the native description.